packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -172,12 +174,17 @@ export default function ResultsTable({
   const [tableCellScale, setTableCellScale] = useState(1);
 
   const gb = useGrowthBook<AppFeatures>();
-  const showTimeSeriesButton =
+  let showTimeSeriesButton =
     baselineRow === 0 &&
     tableRowAxis === "metric" &&
     !disableTimeSeriesButton &&
     gb.isOn("experiment-results-timeseries");
 
+  // Disable time series button for stopped experiments before we added this feature (& therefore data)
+  if (status === "stopped" && endDate <= "2025-04-03") {
```

Comment:
I assume you tested this works; something weird about endDate string comparisons just bugging me :)

Reply:
works for me

Reply:
Yeah -- it works because we use ISO string which is (by design) lexically comparable like this -- but agreed that it can be weird at first glance 👀